src/client.rs

```diff
@@ -1286,6 +1286,61 @@ impl<T: Read + Write> Session<T> {
             .and_then(|lines| parse_ids(&lines, &mut self.unsolicited_responses_tx))
     }
 
+    /// The SORT command is a variant of SEARCH with sorting semantics for
+    /// the results. There are two arguments before the searching
+    /// criteria argument: a parenthesized list of sort criteria, and the
+    /// searching charset.
+    ///
+    /// sort            = ["UID" SP] "SORT" SP sort-criteria SP search-criteria
+    /// sort-criteria   = "(" sort-criterion *(SP sort-criterion) ")"
+    /// sort-criterion  = ["REVERSE" SP] sort-key
+    /// sort-key        = "ARRIVAL" / "CC" / "DATE" / "FROM" / "SIZE" /
+    ///                   "SUBJECT" / "TO"
+    /// search-criteria = charset 1*(SP search-key)
+    /// charset         = atom / quoted
+    ///                     ; CHARSET values MUST be registered with IANA
+    /// sort-data       = "SORT" *(SP nz-number)
```

Comment:
Where does `sort-data` come into play?

Reply:
I _think_ that's only used in the response, so probably not worth including in the docs.

Reply:
:heavy_check_mark: Done